Features/AdminComponentPreview/functions.php

```diff
@@ -37,13 +37,11 @@ function enqueueComponentScripts()
     if (is_user_logged_in() || is_admin()) {
         if (is_admin()) {
             add_action('admin_enqueue_scripts', NS . 'enqueueComponentScripts');
-
             // add image to the flexible content component name
             add_filter('acf/fields/flexible_content/layout_title', function ($title, $field, $layout, $i) {
-                $componentPath = "/Components/$layout[name]/";
-                $componentPreviewDesktopPath = get_template_directory() . $componentPath . 'preview-desktop.jpg';
-                $componentPreviewDesktopUrl = get_template_directory_uri() . $componentPath . 'preview-desktop.jpg';
-
+                $componentPath = "Components/" . ucfirst($layout['name']) . "/";
+                $componentPreviewDesktopPath = Asset::requirePath($componentPath . 'preview-desktop.jpg') ;
+                $componentPreviewDesktopUrl = Asset::requireUrl($componentPath . 'preview-desktop.jpg');
```

Comment:
For consistency, we should probably save the values in variables and interpolate like this `"Components/${componentName}/"`. What do you think?